### DISCUSSION TITLE
Question：ICC Application

### DISCUSSION THREAD

**Discussion by IoriAkatsuki** [seerge/g-helper#1196] [Q&A] [answered]
Some of models in original Armorny Crate is capable with switching built-in icc files selecting different gamut or color spaces for laptops.
Im wondering that if the g-helper can also switch different icc files? If not, is there any other mini tools can handle this problem? Many thanks to all!

**Comment by seerge** [comment, marked as answer]
@IoriAkatsuki hello,

You can add and switch ICC color profiles using windows native Color Management app (just run it from start menu). As soon as profile is added you can also set it from Settings -> Display

I can't add ICC profiles to G-Helper by a very simple reason, I don't have them, and they are different for each model (or at least for each display panel model).

This is a duplicate of this issue btw https://pcmonitors.info/articles/using-icc-profiles-in-windows/

**Reply by IoriAkatsuki** [reply]
Many thanks！
As far as i know, Asus stores all color profile in C:\Windows\System32\spool\drivers\color folder.  Perhaps a shortcut can be added？ I'll try : )

**Reply by seerge** [reply]
It's not Asus storing profiles there, it's windows Color Management. As soon as profile is added and associated with screen you System -> Display is enough to select it

![Screenshot 2023-08-28 115956](https://github.com/seerge/g-helper/assets/5920850/aa4d7b83-aa02-474d-be4e-c854c04cae1e)
